pkg/lintcontext/create_contexts.go

```diff
@@ -113,13 +112,3 @@ func CreateContextsWithOptions(options Options, filesOrDirs ...string) ([]LintCo
 	}
 	return contexts, nil
 }
-
-// CreateContextsFromHelmArchive creates a context from TGZ reader of Helm Chart.
-func CreateContextsFromHelmArchive(fileName string, tgzReader io.Reader) ([]LintContext, error) {
```

Comment:
We can't remove this function, it was added for library use and removing this will break code that uses it. See https://github.com/stackrox/kube-linter/pull/173